# Paremeter settings for Executive Planner

ExecutivePlanner:

# Path settings
  path_filename: paths/test_path.txt    # Name to text file with path coordinates to loa
  path_package: runners_navigation      # Package name that the path text file is in
  pt_spacing: 0.01                      # How far apart points are spaced in the path text file [m]
  planning_distance: -1.5               # How far ahead to generate the plan [m]
  
# Docking
  dock_x: 1.1
  dock_y: 2.2
  dock_yaw: 1.23
